### PR TITLE
Enable cross-platform version

### PR DESCRIPTION
**Pull request by li-kai**
I had some errors trying to get the project to start on windows. So I added the changes (mostly having to deal with line-endings). I also added postcssJs which wasn't installed as a dependency.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# to native line endings on checkout.
+*.c text
+*.h text
+
+# Declare files that will always have LF line endings on checkout.
+*.sln text eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```
Comment (yangshun) [inline .gitattributes]: Add newline

**Comment by yangshun** [comment]
Sorry I forgot about the `postcss-js` part 😂 Have actually added it back about an hour ago. Good to have someone who uses Windows! Traditionally we never had good support for Windows since all of us use Mac. Feel free to merge this PR yourself.

**Comment by yangshun** [comment]
@li-kai Have added you to our nusmodifications organization. Feel free to push your own PR branch to this remote instead, it makes it easier for each of us to checkout PR branches and try (:

**Comment by yangshun** [comment]
Just checkout-ed the PR and tested on my Mac. Works well 👍

**Comment by li-kai** [comment]
@yangshun Sorry still a bit new to contributing. How do I push my PR to this remote?

**Comment by yangshun** [comment]
For this PR, just merge it as it is. For future PRs, create a branch on this remote directly. I believe you have the write permissions for it.

**Pull request merged**